plugins/widget/plugin.js

```diff
@@ -3570,13 +3570,16 @@
 			// ENTER.
 			if ( keyCode == 13 ) {
 				widget.edit();
-				// CTRL+C or CTRL+X.
+			// CTRL+C or CTRL+X.
 			} else if ( keyCode == CKEDITOR.CTRL + 67 || keyCode == CKEDITOR.CTRL + 88 ) {
 				copySingleWidget( widget, keyCode == CKEDITOR.CTRL + 88 );
 				return; // Do not preventDefault.
-			} else if ( keyCode in keystrokesNotBlockedByWidget || ( CKEDITOR.CTRL & keyCode ) || ( CKEDITOR.ALT & keyCode ) ) {
-				// Pass chosen keystrokes to other plugins or default fake sel handlers.
-				// Pass all CTRL/ALT keystrokes.
+			// Pass chosen keystrokes to other plugins or default fake sel handlers.
+			// Pass all CTRL/ALT and SHIFT + F10 keystrokes.
+			} else if ( keyCode in keystrokesNotBlockedByWidget ||
+				( keyCode == CKEDITOR.SHIFT + 121 ) ||
```

Comment:
I would be for utilising existing mechanism for not blocked keystrokes, so `keystrokesNotBlockedByWidget` could be modified by adding there this keystroke too:

```js
// LEFT, RIGHT, UP, DOWN, DEL, BACKSPACE - unblock default fake sel handlers.
var keystrokesNotBlockedByWidget = { 37: 1, 38: 1, 39: 1, 40: 1, 8: 1, 46: 1 }; // Existing code.

// Do not block SHIFT + F10 which opens context menu (#1901).
keystrokesNotBlockedByWidget[ CKEDITOR.SHIFT + 121 ] = 1; // Added code.
```

and then this code modification is not needed here.